improve documentation grammar and update external links (#878)
Consensys/doc.linea: docs/get-started/tooling/analytics/mobula.mdx

image: /img/socialCards/mobula.jpg
---

> [Mobula](https://docs.mobula.io) serves as a specialized data layer offering analytics 

Reviewer: ```suggestion
[Mobula](https://docs.mobula.io) is a specialized data layer offering analytics 
```

good spot, but "serves as" is unnecessarily complex to begin with.
